stytch/shared/jwt_helpers.py

```diff
@@ -33,7 +34,13 @@ def authenticate_jwt_local(
     comparing timestamps. It defaults to zero.
     """
     jwt_audience = project_id
-    jwt_issuer = f"stytch.com/{project_id}"
+    default_issuer = f"stytch.com/{project_id}"
+    allowed_issuers: List[str] = [
+        default_issuer,
+        base_url[
+            :-1
+        ],  # Remove trailing slash, our issuers will never have a trailing slash
+    ]
```

Comment:
We can make this a lot cleaner to read
```suggestion
    allowed_issuers: List[str] = [default_issuer, base_url.rstrip("/")]
```

Reply:
Yeah the python formatting from codegen decided this was the best format lol

But if we change to use `rstrip` this should work 👍